modules/ngx_http_cbor_ic_modules/ngx_http_cbor_resp_ic_module.c

```diff
@@ -160,10 +160,6 @@ typedef enum
 static consume_result_t
 consume_body(ngx_pool_t *p, ngx_chain_t *bufs, buf_t *buf)
 {
-    // Skip in-file buffers
-    if (bufs->buf->in_file)
-        return CONSUME_EINFILE;
```

Comment:
I am actually not sure why the check was here, do we know who wrote it?

If I am allowed to guess, maybe "in_file" means it is big in size. Are we sure we want to copy a big file to in-memory buffer?

Reply:
Yes, that is my concern as well. Perhaps we could put a limit on maximum body size we'd be willing to read.

That said, in my tests (on a dev boundary node) I saw _all_ responses be in files, regardless of their size (e.g even a 300 byte response).

I posted a question in the Nginx Slack asking whether this is always the case or whether there is some nginx config field that can be set, similar to `client_body_buffer_size`, since I did not find one in the docs.